source/nodejs/adaptivecards/src/strings.ts

```diff
@@ -1,37 +0,0 @@
-export class Strings {
-    static readonly errors = {
-        unknownElementType: (typeName: string) => `Unknown element type "${typeName}". Fallback will be used if present.`,
-        unknownActionType: (typeName: string) => `Unknown action type "${typeName}". Fallback will be used if present.`,
-        elementTypeNotAllowed: (typeName: string) => `Element type "${typeName}" is not allowed in this context.`,
-        actionTypeNotAllowed: (typeName: string) => `Action type "${typeName}" is not allowed in this context.`,
-        invalidPropertyValue: (value: any, propertyName: string) => `Invalid value "${value}" for property "${propertyName}".`,
-        showCardMustHaveCard: () => `"An Action.ShowCard must have its "card" property set to a valid AdaptiveCard object.`,
-        invalidColumnWidth: (invalidWidth: string) => `Invalid column width "${invalidWidth}" - defaulting to "auto".`,
```

Comment:
Definitely dont check this in :) There's a bunch of deletes here which i am not sure is what we want!

Reply:
I resolved these conflicts :)